index.html

```diff
@@ -1,68 +1,286 @@
+    <section class="about">
+      <div class="about-me flex column">
+        <div class="is-me flex column">
+          <h2 class="about-text">About <br />Myself</h2>
+          <p class="self">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+
+          <div class="intro-items">
+            <a href="#">LET'S CONNECT</a>
+            <div class="intro-connect">
+              <ul class="list-items skill-items">
+                <li>
+                  <a href="#"> <img src="images/github.png" alt="" /> </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img src="images/Linkedin icon.svg" alt="" />
+                  </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/angellist.png" alt="" /> </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/twitter.png" alt="" /> </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/medium.png" alt="" /> </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="knowdge flex column">
+          <ul>
+            <li>
+              <div class="know flex column">
+                <p class="know-text flex">
+                  <span class="text">Languages</span>
+                  <span class="know-line">></span>
+                </p>
+                <ul class="know-sec flex column">
+                  <li class="know-list flex">
+                    <img src="images/javaScript.png" alt="js logo" />
+                    <p class="job-text">javaScript</p>
+                  </li>
+                  <li class="know-list flex">
+                    <img src="images/html.png" alt="js logo" />
+                    <p class="job-text">HTML</p>
+                  </li>
+                  <li class="know-list flex">
+                    <img src="images/css.png" alt="js logo" />
+                    <p class="job-text">CSS</p>
+                  </li>
+                </ul>
+              </div>
+            </li>
+            <li>
+              <div class="know">
+                <p class="know-text flex">
+                  <span class="text">Frameworks</span
+                  ><span class="arrow">></span>
+                </p>
+              </div>
+            </li>
+            <div class="arrow-line">
+              <hr />
+            </div>
+            <li>
+              <div class="know">
+                <p class="know-text flex">
+                  <span class="text">Skills</span><span class="arrow">></span>
+                </p>
```

Comment:
[OPTIONAL] I think the parent tag here should be div and not p tag. The <p> tag defines a paragraph  while the DIV tag is simply an HTML container. Its only purpose is to let you group together other document elements to which to apply the same CSS style